src/components/App/App.css

```diff
@@ -0,0 +1,8 @@
+.wrap {
+  margin: auto;
+  width: 300px;
+}
+
+.wrap__title {
+  font-family: Arial, Helvetica, sans-serif;
+}
```

Comment:
Codacy found an issue: [Unexpected missing end-of-source newline (no-missing-end-of-source-newline)](https://app.codacy.com/gh/WebHeroSchool/evafinal/pullRequest?prid=5877807)